nilearn/decomposition/canica.py

```diff
@@ -18,7 +18,7 @@
 
 @fill_doc
 class CanICA(_MultiPCA):
-    """Perform Canonical Independent Component Analysis [1]_ [2]_.
+    """Perform Canonical Independent Component Analysis :footcite:`Varoquaux2010c` :footcite:`Varoquaux2010d`.
```

Comment:
CI says this line is too long:
```suggestion
    """Perform Canonical Independent Component Analysis \
    :footcite:`Varoquaux2010c` :footcite:`Varoquaux2010d`.
```
https://github.com/nilearn/nilearn/actions/runs/4314198218/jobs/7526922367#step:5:16

Reply:
> CI says this line is too long:
> 
> https://github.com/nilearn/nilearn/actions/runs/4314198218/jobs/7526922367#step:5:16

Thank you for reviewing! The fix is committed